.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+<!--- IF YOU REMOVE THIS TEMPLATE WE WILL CLOSE YOUR PULL REQUEST WITHOUT RESPONSE.
+Just read it.
+First of all, thank you for contributing to our project. 
+Please make sure to fill out the form below so we can get a glimpse of what this PR does. -->
+_Add your summary here._
+<What has been changed?
+<Why has it been changed?
+<What has been added?
+<What has been fixed?
+<What needs to be tested?
+<Do you have any GIFs to attach?>
+
+__General:__
+
+Issue: <Jira / Github Issue>
+
+Dependencies:
+
+__Reviewer__:
```

Comment:
You can remove the part form General to Reviewer, and replace it with: Dependencies.
Also Changelog may not be so important for a pr.